pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+norecursedirs=venv_py2 venv_py3
```

Comment:
This excludes the virtual environment folders you need to create when running the tests, from actually being included in the scan for tests.
I guess it's actually just saying don't recurse through these, but the effect is that it won't find any tests directly in those folders anyway. `--ignore` pytest arg didn't appear to have any effect for me.

Reply:
If you run pytest from the tests folder outside of Houdini, then it will error saying it can't import Hou. 
And as the README instructions state, you must point the env var to the test folder in order for the tests to run within Houdini, so in that situation, yes the ini file is picked up.